src/components/textField/validators.ts

```diff
@@ -8,9 +8,15 @@ const priceRegEx = /^[0-9]{1,9}([.][0-9]{1,2})?$/;
 
 const validators = {
   required: (value = '') => !_.isEmpty(value),
-  email: (value = '') => new EmailValidator().isValid(value),
+  email: (value = '') => {
```

Comment:
I think it could be a breaking change for other users.
I think we should add a new validation `optionalEmail` that will validate this case (empty string) and leave the `email` validator as it is.